src/command.ts

```diff
@@ -281,8 +281,8 @@ export class Command {
       setupLoggers();
     }
 
-    if (getInheritedOption(options, "config")) {
-      options.configPath = getInheritedOption(options, "config");
+    if (getInheritedOption(options, "configPath")) {
```

Comment:
I see some rearranging of config. Does this affect the way the CLI works? I think it might. I think the name of the option that you can type in the CLI is "--config" (search for it [here](https://firebase.google.com/docs/cli#the_firebasejson_file)) and this is honoring the way it comes in as user input. I don't fully understand the CLI but I think that this might mean the user has to type `--configPath` to do the same thing which is kind of a breaking change.

Reply:
Good point I should have opened this for discussion - I was getting an undefined error by this, likely because this makes assumptions about the structure of config vs configPath. I assumed at the time that this was just a typo mistake that hadn't been caught yet. I can't find anything on the internet about it

@joehan do you have any thoughts?

Reply:
Nvm reverting this and making it work :)